staker/block_challenge_backend.go

```diff
@@ -71,6 +73,14 @@ func NewBlockChallengeBackend(
 	}, nil
 }
 
+func (b *BlockChallengeBackend) SetDebugEspressoIncorrectHeight(h uint64) {
```

Comment:
maybe put some prefix to this to make it clear this is a debug function, something like EspressoDebug_, and perhaps put it below the standard functions with a comment